test/Transactions/Orleans.Transactions.Azure.Test/TransactionConcurrencyTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit.Abstractions;
+using Xunit;
+using Orleans.Transactions.Tests;
+
+namespace Orleans.Transactions.AzureStorage.Tests
+{
+    [TestCategory("Azure"), TestCategory("Transactions"), TestCategory("Functional")]
+    public class TransactionConcurrencyTests : TransactionConcurrencyTestRunner, IClassFixture<TestFixture>
+    {
+        public TransactionConcurrencyTests(TestFixture fixture, ITestOutputHelper output)
+            : base(fixture.GrainFactory, output)
+        {
```

Comment:
I think you aren't skiiping azure tests correctly. those are still run by Jenkins and failed

Reply:
This was by design.  I only intended to skip the memory test, because those were the only one's failing locally.  Azure tests passed for me.  This simplifies things, as I don't need to do this somewhat kludgy skip method, but rather just mark all the test as skip.  Will change that.

Bumbed these tests aren't passing.  Had hoped all was well and there was just some issue with the memory storage.

Reply:
Oh, I'm dork.. I see what you mean now.. NM